### PR TITLE
Add r-spscomps

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by dpryan79** [comment]
@conda-forge/staged-recipes This is ready for review. It's not compatible with R 3.6

File: recipes/r-spscomps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    - r-stringr
+
+test:
+  commands:
+    - $R -e "library('spsComps')"           # [not win]
+    - "\"%R%\" -e \"library('spsComps')\""  # [win]
+
+about:
+  home: https://github.com/lz100/spsComps
+  license: GPL-3.0-only
```
Comment (cbrueffer) [inline recipes/r-spscomps/meta.yaml:62]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/r-spscomps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/spsComps_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/spsComps/spsComps_{{ version }}.tar.gz
+  sha256: 1d7e359ef62ffb543d0d0221130be1788f9654073599bf526fb52ffd7a348e0b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  skip: true  # [r-base < 4]
```
Comment (cbrueffer) [inline recipes/r-spscomps/meta.yaml:19]: Does `r-base` work here? Usually we use `r_base`.

Comment (dpryan79) [reply recipes/r-spscomps/meta.yaml:19]: I had tried that and it doesn't (I was surprised)

Comment (dpryan79) [reply recipes/r-spscomps/meta.yaml:19]: I take that back, apparently nothing was built

File: recipes/r-spscomps/meta.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ source:
 build:
   merge_build_host: True  # [win]
   number: 0
-  skip: true  # [r_base < 4]
+  skip: true  # [r_base < '4']
```
Comment (cbrueffer) [inline recipes/r-spscomps/meta.yaml:19]: Yeah, what I usually use is `[r_base == "3.6"]`; what you use above should work.

Comment (dpryan79) [reply recipes/r-spscomps/meta.yaml]: Maybe the double quotes helps? I'll give it a try

Comment (cbrueffer) [reply recipes/r-spscomps/meta.yaml]: It may the ´<´ actually, not sure that ever worked for me.

Comment (dpryan79) [reply recipes/r-spscomps/meta.yaml]: I guess that's the trick

**Comment by dpryan79** [comment]
@conda-forge/staged-recipes This is actually ready for review now

**Pull request merged**